completion/services.py

```diff
@@ -103,6 +103,24 @@ def get_completable_children(self, node):
             user_children = [node]
         return user_children
 
+    @staticmethod
+    def matches_vertical_optional_completion(optional_vertical, optional_child):
+        """
+        Checks if child should count towards a vertical's completion.
```

Comment:
Could we expand this docstring a bit to explain the concept of "optional" XBlocks?

Reply:
done